module_1/src/com/epam/cycles/CyclesTaskD.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.cycles;
+import java.math.BigInteger;
+
+public class CyclesTaskD {
+
+    private static final int AMOUNT_OF_NUMBERS = 200;
+
+    public static void main(String[] args) {
+        BigInteger product = new BigInteger("1");
```

Comment:
It is funny but product isn't a composition of numbers, product is an article or substance that is manufactured or refined for sale :) Please rename the variable
```suggestion
        BigInteger composition = new BigInteger("1");
```

Reply:
But i've really found that - product noun (IN MATHEMATICS) the result you get when two or more numbers are multiplied together.

Reply:
> But i've really found that - product noun (IN MATHEMATICS) the result you get when two or more numbers are multiplied together.

Okay, if it is correct, no problem, just naming to be more clear for reader :)